_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -2302,6 +2302,30 @@ mfxStatus ImplementationAvc::AsyncRoutine(mfxBitstream * bs)
 
             if (newTask.m_picStruct[ENC] == MFX_PICSTRUCT_FIELD_BFF)
                 std::swap(newTask.m_type.top, newTask.m_type.bot);
+
+            mfxU64 timeStamp = newTask.m_timeStamp;
```

Comment:
Please turn low level code below to function. Let's fight for lower size and better readability of [ImplementationAvc::AsyncRoutine(..)].

Reply:
Ok. It is better to turn them to function.